streaming/storage/subscription.go

```diff
@@ -67,3 +69,49 @@ func (sub *Subscription) Close() error {
 		return err
 	}
 }
+
+func ConcatSubscriptions(ctx context.Context, subs ...*Subscription) *Subscription {
+	count := len(subs)
+
+	channels := make([]reflect.SelectCase, len(subs)*2+1)
```

Comment:
Ok this kind of length of slice magic deserves at least a small comment about what one is trying to achieve ;)

Reply:
also, you have just calculated len(subs) 2 lines above so you can use 'count' variable here instead of calculating length once more